squad/frontend/templates/squad/base.jinja2

```diff
@@ -33,8 +33,7 @@
         </a>
         <div class="navbar-header">
           <a class="navbar-brand" href="/">
-            <span class="fa fa-line-chart" aria-hidden="true"></span>
-            {{squad_site_name()}}
+            <img style="height: 27px;" src="{{static('squad_sign.svg')}}">
```

Comment:
please add alt='{{squad_site_name()}}' to the img tag for accessibility

today qa-reports shows "Linaro QA" instead of "SQUAD" (via squad_site_name(), defined in the configuration). this makes it display the SQUAD logo unconditionally.

Reply:
oups, I though I had alt there. 
I know it displays the logo only without any site name. I guess that's OK. I thought about making the image configurable. Maybe this is an option?